src/components/PatchAlbum.jsx

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState, useEffect } from 'react';
+import { Deserializer } from 'jsonapi-serializer';
+import * as Yup from 'yup';
+import { Formik, Form, Field } from 'formik';
+import { useNavigate, useParams } from 'react-router-dom';
+import config from '../config';
+import useAuth from '../hooks/useAuth';
+
+export default function PatchAlbum() {
+  const { id } = useParams();
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState('');
+  const [error, setError] = useState(false);
+  const [artists, setArtists] = useState([]);
+  const [album, setAlbum] = useState(null);
+  const { currentUser } = useAuth();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setLoading(true);
+    fetch(`${config.API_URL}/api/artists`)
+      .then((response) => {
+        if (!response.ok) {
+          setError(true);
+          return [];
```

Comment:
Misma sugerencia que antes 
```suggestion
          return response.text().then((message) => Promise.reject(new Error(message)));
```